lib/beyond_api/concerns/connection.rb

```diff
@@ -32,6 +32,27 @@ def delete(path, params = {})
         handle_request { agent.delete(path, parse_request(params)) }
       end
 
+      def upload_file(path, file_path, params = {})
+        handle_request do
+          agent.post(path) do |request|
+            request.headers['Content-Type'] = Utils.file_content_type(file_path)
```

Comment:
Do we keep the content type logic in the `Utils` file? or should we use smth already implemented? for example, the `mime-types` gem?